superset-frontend/src/components/ReportModal/HeaderReportDropdown/index.tsx

```diff
@@ -147,61 +135,61 @@ export default function HeaderReportActionsDropDown({
       </Menu.Item>
     </Menu>
   );
-
   return (
-    canAddReports() && (
-      <>
-        <ReportModal
-          userId={user.userId}
-          showModal={showModal}
-          onHide={() => setShowModal(false)}
-          userEmail={user.email}
-          dashboardId={dashboardId}
-          chart={chart}
-        />
-        {report ? (
-          <>
-            <NoAnimationDropdown
-              // ref={ref}
-              overlay={menu()}
-              trigger={['click']}
-              getPopupContainer={(triggerNode: any) =>
-                triggerNode.closest('.action-button')
-              }
+    <>
+      {canAddReports() && (
```

Comment:
since this is a functional component we can probably make this a prop instead of a method. We can fix this on the merge to master PR.